amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/impl/PersistentQueue.java

```diff
@@ -53,13 +56,15 @@
     @Getter
     private PersistentTopic indexTopic;
 
-    private ObjectMapper jsonMapper = ObjectMapperFactory.create();
+    private ObjectMapper jsonMapper;
 
-    public PersistentQueue(String queueName, PersistentTopic indexTopic, long connectionId,
+    public PersistentQueue(String queueName, PersistentTopic indexTopic,
+                           long connectionId,
                            boolean exclusive, boolean autoDelete) {
         super(queueName, true, connectionId, exclusive, autoDelete);
         this.indexTopic = indexTopic;
         topicNameValidate();
+        jsonMapper = new ObjectMapper();
```

Comment:
here, it should be "this.jsonMapper = new ObjectMapper()";

Reply:
OK.